public/keymaps/p/ploopyco_mouse_default.json

```diff
@@ -0,0 +1,18 @@
+{
+    "keyboard": "ploopyco/mouse",
+    "keymap": "default",
+    "commit": "fbdc65e2e956e9d51c94fecdbce1f0abc2817950",
+    "layout": "LAYOUT",
+    "layers": [
+        [
+            "LCTL(KC_C)",
+            "KC_BTN1",
+            "KC_BTN3",
+            "KC_BTN2",
+            "LCTL(KC_V)",
```

Comment:
This might be a mistake on the `qmk_firmware` keymap, but that has `LCTL(KC_C)` here.

https://github.com/qmk/qmk_firmware/blob/76e35e02cc6caf88eb5bb06c9780619b78725d16/keyboards/ploopyco/mouse/keymaps/default/keymap.c#L22

Reply:
Yeah, that's a mistake. One side is supposed to be c, and the other v.

Reply:
Okay then.